roles/download/defaults/main.yml

```diff
@@ -115,7 +115,15 @@ flannel_cni_version: "v1.1.0"
 cni_version: "v1.1.1"
 weave_version: 2.8.1
 pod_infra_version: "3.7"
+
 cilium_version: "v1.12.1"
+# Enable Hubble without install
+cilium_enable_hubble: false
+# Enable Hubble install
+cilium_hubble_install: false
+# Enable auto generate certs if cilium_hubble_install: true
```

Comment:
s/`if cilium_hubble_install: true`/`if cilium_enable_hubble: true`/ according to line 1239?